stories/modal/index.stories.tsx

```diff
@@ -1,10 +1,22 @@
 import { storiesOf } from "@storybook/react"
 
-import generateStory from "./generator"
+import generateModalStory from "./generator"
 
 storiesOf("Modal", module)
-  .add("Small", generateStory({ size: "small" }))
-  .add("Medium", generateStory({ size: "medium" }))
-  .add("Large", generateStory({ size: "large" }))
-  .add("Fulscreen", generateStory({ size: "fullscreen" }))
-  .add("Dark Fulscreen", generateStory({ style: "dark", size: "fullscreen" }))
+  .add("Small", generateModalStory({ size: "small" }))
+  .add(
+    "Small with onClose event",
+    generateModalStory({
+      size: "small",
+      onClose: () => {
+        alert("onClose callback called")
```

Comment:
Could you use `action` from [actions add on](https://www.npmjs.com/package/@storybook/addon-actions) instead? It will then pop-up in the action tab.

Check out how it's used in in the button story: https://github.com/carforyou/carforyou-components-pkg/blob/master/stories/button.stories.js#L9

Reply:
Nice wasn't aware of it though like what's the most recognizable way to showcase the callback was executed 😆